book/image-formats.rst

```diff
@@ -1,72 +1,88 @@
 Configure image formats
 =======================
 
-If you are using images, you probably also care about the available image
-formats. Sulu also supports you with that issue. You can define different image
-formats, which Sulu will create for every uploaded image. This generation
-is based on your configuration. You can define the formats in the file
-`Resources/themes/<theme>/config/image-formats.xml` or
-`%kernel.root_dir%/config/image-formats.xml` (if theme are not enabled).
-Take a look at the following file for an example:
+To show images on your website you should define different image formats for optimized output.
+
+Image formats can be defined in:
+
+ - `app/config/image-formats.xml`
+
+Or when you use the SuluThemeBundle you can define the formats in your theme folder:
+
+ - `path/to/<theme>/config/image-formats.xml`
+
+The following example show you different ways how you can define image formats:
 
 .. code-block:: xml
 
     <?xml version="1.0" encoding="UTF-8"?>
     <formats xmlns="http://schemas.sulu.io/media/formats"
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="http://schemas.sulu.io/media/formats http://schemas.sulu.io/media/formats-1.1.xsd">
-        <format key="640x480">
+
+        <!-- Fixed width and dynamic height -->
+        <format key="300x">
+            <scale x="300"/>
+        </format>
+
+        <!-- Dynamic width and fixed height -->
+        <format key="x200">
+            <scale y="200"/>
+        </format>
+
+        <!-- Fixed width and fixed height -->
+        <format key="300x200">
+            <scale x="300" y="200"/>
+        </format>
+
+        <!-- Max width and max height -->
+        <format key="300x200-inset">
+            <scale x="300" y="200" mode="inset"/>
+        </format>
+    </formats>
+
+Defining meta title
+-------------------
+
+Content manager can crop an image in the backend you should add a meta title to every image format
```

Comment:
`You should define a meta title for every image format, since these titles are shown to the content manager in the administration interface when cropping the images.`